lib/cisco_node_utils/bgp_af.rb

```diff
@@ -98,21 +98,41 @@ def set_args_keys(hash={}) # rubocop:disable Style/AccessorMethodName
     #                      PROPERTIES                      #
     ########################################################
 
+    # IOS XR?
+    def ios_xr?
+      platform == :ios_xr
+    end
+
+    def vrf_unsupported_xr?
+      ios_xr? && @vrf != 'default'
+    end
+
+    def fail_vrf_unsupported_xr(reason)
```

Comment:
Nit: rename method to `fail_if_vrf_unspported_xr` so it's clear that the method does not always call the `fail` method